### PR TITLE
Feat/upcoming movie

### PR DESCRIPTION
**Pull request by MahdiHosseinizade**

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
     <section class="footer" id="footer">
-        <a href="" class="logo">
+        <a  class="logo">
```
Comment (amRashidi) [inline src/components/Footer.vue:3]: href??

File: src/components/Movies.vue [hunk 2]
```diff
@@ -5,94 +5,51 @@
...
-      {
-        name: 'Brad pit',
-        image: bradpit
-      },]
+
+  const options = {
+  method: 'GET',
+  headers: {
+    accept: 'application/json',
+    Authorization: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI1YTU2YzhmYzk2N2EzMTA1NDk2OTA3MmFhY2E2MDVmNiIsInN1YiI6IjY0YTUxYjMyMWJmMjY2MDEwNTE4MDAyMiIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.HxmHxMHAYsrdt_Q3S613yYxLtOJCHd2gYTqAfOAIY2I'
```
Comment (amRashidi) [inline src/components/Movies.vue]: avoid hardcode token like this!!

File: src/components/Movies.vue [hunk 3]
```diff
@@ -5,94 +5,51 @@
...
+  }
+};
+
+const getUpcomingMovie = async () =>{
+  const respponse = await fetch('https://api.themoviedb.org/3/movie/upcoming?language=en-US&page=1', options)
+  const data = await respponse.json();
+  const upcomingMovie = data.results.slice(0, 4);
```
Comment (amRashidi) [inline src/components/Movies.vue]: add guard (what if data length is lower than 4?)

**Review by amRashidi — CHANGES_REQUESTED** [review]

File: src/components/Footer.vue (hunk 1 above)
Comment (MahdiHosseinizade) [reply src/components/Footer.vue:3]: I remove this

File: src/components/Movies.vue (hunk 2 above)
Comment (MahdiHosseinizade) [reply src/components/Movies.vue]: Thank you for your comment and guidance regarding avoiding hardcoding the token. I appreciate your input, and I have made the necessary changes to improve the code.
Following your suggestion, I have moved the API base URL and access token to the "constants" folder.

File: src/components/Movies.vue (hunk 3 above)
Comment (MahdiHosseinizade) [reply src/components/Movies.vue]: I remove the slice and now show all the data

**Pull request merged**